web/src/pages/Events.jsx

```diff
@@ -42,56 +47,90 @@ export default function Events() {
     onClose: editEventOnClose,
   } = useDisclosure();
 
+  function setNewSearch(newSearchText) {
+    setSearch(newSearchText);
+  }
+
+  useEffect(() => searchRef.current?.focus(), []);
+
   return (
     <>
-      <Tabs
-        colorScheme='green'
-        variant='solid-rounded'
-        isFitted
-        isLazy
-        isManual
-      >
-        <TabList>
-          <Tab>{tabTitle?.one}</Tab>
-          <Tab>{tabTitle?.two}</Tab>
-          <Tab>{tabTitle?.three}</Tab>
+      <VStack spacing={6}>
+        <HStack w='100%'>
+          <Input
+            ref={searchRef}
+            type='text'
+            placeholder='Search Events'
+            colorScheme='green'
+            value={searchText}
+            onChange={e => setSearchText(e.target.value)}
+            onKeyDown={async e => {
+              if (e.key === 'Enter') setNewSearch(searchText);
+            }}
+          />
           <IconButton
-            aria-label='add an event'
-            icon={<MdAdd fontSize='1.25rem' />}
             colorScheme='green'
-            bgColor={tabColor}
-            borderRadius='full'
+            icon={<MdSearch fontSize='1.25rem' />}
+            onClick={() => setNewSearch(searchText)}
+          />
+          <IconButton
+            colorScheme='green'
+            variant='outline'
+            icon={<MdAdd fontSize='1.25rem' />}
             onClick={addEventOnOpen}
```

Comment:
There are two addNewEvent button, can you please update that?

Reply:
removed duplicate button